src/main/java/History.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+import com.google.sps.data.VisualizationData;
```

Comment:
I think this can be removed since it's in the same package as this class

Reply:
I thought you had to import it but you're right, it compiles no problem.